test/e2e/cluster_upgrade.go

```diff
@@ -522,9 +522,9 @@ func migTemplate() (string, error) {
 		// An `instance-groups managed describe` call outputs what we want to stdout.
 		output, _, err := retryCmd("gcloud", "compute", "instance-groups", "managed",
 			fmt.Sprintf("--project=%s", testContext.CloudConfig.ProjectID),
-			fmt.Sprintf("--zone=%s", testContext.CloudConfig.Zone),
 			"describe",
-			testContext.CloudConfig.NodeInstanceGroup)
+			testContext.CloudConfig.NodeInstanceGroup,
```

Comment:
maybe put the --zone before the instance group? just in case the flag parsing changes at some point and it starts deciding "--zone" is also the name of an instance group.